.github/workflows/build.yaml

```diff
@@ -255,7 +255,8 @@ jobs:
           token: ${{github.token}}
       - name: Install cdt Packages
         run: |
-          apt install -y ./*.deb
+          apt-get update
```

Comment:
there is already another `apt-get update` in the same job earlier, but only for the `deb-install` case
https://github.com/AntelopeIO/leap/blob/9ff0583b33259b965bd11f3bc228cd2a30e7c814/.github/workflows/build.yaml#L236-L239
Could consider a single step near the beginning of the job that always does the `apt-get update`, to avoid situations where it's done twice in the same job, and avoid the duplication.

Reply:
Good idea. Done.  Only implemented a new step in the `libtester-tests` job as the other looked currently to be and continue to be a single use scenario.